int32 antennaCommand
int32 antennaPan
